src/Foundation/Features/Pages/StandardPage/Index.cshtml

```diff
@@ -22,6 +22,11 @@
 <div class="row">
     <div class="col-12 standard-page__container">
         <div class="standard-page__hero @heroClass">
+            <div class="row">
```

Comment:
It would be good to verify that content is not empty before rendering a html markup